### ISSUE TITLE
pickle NPC mission script

### ISSUE THREAD

**Issue by minipunch** [minipunch/usarealismrp-fxserver#1829]
https://store.picklemods.com/package/5376679

@Zeeroqt said he'd work on it

**Comment by Zeeroqt** [comment]
yep

**Comment by Zeeroqt** [comment]
callback system needs to be fixed and its all set

**Comment by minipunch** [comment]
Do we have an update on this @Savage912-Official ? Should we close for now ? Too much effort?

**Comment by Savage912-Official** [comment]
Yeah ill take a look today just need to get the callbacks working and we good.

**Comment by Zeeroqt** [comment]
Savage sent me a update to see if it works with the callbacks. I’ll try it once I get home.

**Comment by Zeeroqt** [comment]
okay so its does work but savage tried it with the callback system he uses. Ive made some missions and we already got 2 callback systems but if u wanted to add this callback system for the script i can go ahead and make the pr

**Comment by Zeeroqt** [comment]
the other 2 callback systems for usa dont work with the npc script
